libraries/RC_Channel/RC_Channel.cpp

```diff
@@ -316,6 +316,10 @@ bool RC_Channel::in_trim_dz()
 void RC_Channel::set_override(const uint16_t v, const uint32_t timestamp_us)
 {
     last_override_time = timestamp_us != 0 ? timestamp_us : AP_HAL::millis();
+    if(v == UINT16_MAX)
```

Comment:
The style being used here is incorrect; braces should be on the same line as the if, should be a space after `if`.